nifi-nar-bundles/nifi-azure-bundle/nifi-azure-services-api/src/main/java/org/apache/nifi/services/azure/storage/AzureStorageCredentialsDetails.java

```diff
@@ -22,17 +22,24 @@
 
     private final String storageAccountName;
 
+    private final String storageSuffix;
+
     private final StorageCredentials storageCredentials;
 
-    public AzureStorageCredentialsDetails(String storageAccountName, StorageCredentials storageCredentials) {
+    public AzureStorageCredentialsDetails(String storageAccountName, String storageSuffix, StorageCredentials storageCredentials) {
```

Comment:
Since this is in an API NAR and it's a public constructor we have to leave the old one too. I'd suggest marking the two parameter one `@Deprecated` and having it call the new one with `.blob.core.windows.net` so it has the same behavior.

Reply:
done. Thanks for the tip.